cli/src/pcluster/models/cluster.py

```diff
@@ -193,6 +188,29 @@ def config(self) -> BaseClusterConfig:
     def config(self, value):
         self.__config = value
 
+    @property
+    def config_presigned_url(self) -> str:
+        """Return a pre-signed Url to download the config from the S3 bucket."""
+        return self.bucket.get_config_presigned_url(config_name=self._s3_artifacts_dict.get("source_config_name"))
+
+    @property
+    def compute_fleet_status(self) -> Optional[ComputeFleetStatus]:
+        """Status of the cluster compute fleet."""
+        compute_fleet_status_manager = ComputeFleetStatusManager(self.name)
+        status = compute_fleet_status_manager.get_status()
+        if not status:
+            stack_status_to_fleet_status = {
+                "CREATE_IN_PROGRESS": ComputeFleetStatus.STARTING,
+                "DELETE_IN_PROGRESS": ComputeFleetStatus.STOPPING,
+                "CREATE_FAILED": ComputeFleetStatus.STOPPED,
+                "ROLLBACK_IN_PROGRESS": ComputeFleetStatus.STOPPING,
+                "ROLLBACK_FAILED": ComputeFleetStatus.STOPPED,
+                "ROLLBACK_COMPLETE": ComputeFleetStatus.STOPPED,
+                "DELETE_FAILED": ComputeFleetStatus.STOPPING,
+            }
```

Comment:
would it be better to return a status that signifies it's unknown?

Reply:
good idea